BCStudentSoftwareDevTeam/celts
# Refactored SLC Occurrence across application

### **Fixes Issue #642** 

### Changes Made
-  Changed Course Occurrence toggle to be an input field 
-  Changed the Couse Occurrence input title to be "How often will this course occur?"
-  Updated the unit tests to account for the model change
-  Updated model to account for a new character field
--------------------------------------------------------------------------------------------------------------------------------------------
### How To Test

1. In the terminal in the CELTS directory, enter **database/reset_database.sh test**
2. In the terminal in the CELTS directory, enter **flask run**
3. In the sidebar, navigate to the **Course Proposals** page
4. Create a new proposal (make sure to enter text in the Occurrence input field)
5. Click **Submit** but do **NOT** approve
6. Navigate to the **Service-Learning** section in the sidebar 
7. Click the **Review Button** in your newly created proposal 
8. **_Behold the greatness_**
